Makefile

```diff
@@ -83,7 +84,11 @@ curl-container:
 
 run-curl-container:
 	@echo 'Example: curl -v -s -k -H "Authorization: Bearer `cat /var/run/secrets/kubernetes.io/serviceaccount/token`" https://kube-rbac-proxy.default.svc:8443/metrics'
-	kubectl run -i -t krp-curl --image=quay.io/brancz/krp-curl:v0.0.2 --restart=Never --command -- /bin/sh
+	kubectl run -i -t --rm krp-curl --image=quay.io/brancz/krp-curl:v0.0.2 --restart=Never --command -- /bin/sh
```

Comment:
separate commit

Reply:
Just to clarify... since you want the cert/keys generated dynamically during the tests then I'll have to remove this entirely since it won't be possible to mount the tokens or certificates as they won't exist in the repo.  Did I understand your other comments correctly?

Reply:
Will do.  Thanks!

Reply:
done